lib/event_hub/handler.rb

```diff
@@ -0,0 +1,20 @@
+class EventHub::Handler
+  attr_reader :message
+
+  def self.version(version = nil)
```

Comment:
maybe it also makes sense to add `event` and validate that there is no bug in the config and the handler checks the correct event